test/mlir/conversion/onnx_to_tosa/NN/MaxPoolSingleOut.mlir

```diff
@@ -9,8 +9,8 @@ func.func @test_default_maxpoolsingleout(%arg0 : tensor<5x5x32x32xf32>) -> tenso
 // CHECK-DAG:   "tosa.const"() {value = dense<[0, 2, 3, 1]> : tensor<4xi64>} : () -> tensor<4xi64>
 // CHECK-DAG:   %[[TRANS_ARG:.*]] = "tosa.transpose"(%arg0, %0) : (tensor<5x5x32x32xf32>, tensor<4xi64>) -> tensor<5x32x32x5xf32>
 // CHECK-DAG:   %[[MPOOL_RES:.*]] = "tosa.max_pool2d"(%[[TRANS_ARG]]) {kernel = [3, 3], pad = [0, 0, 0, 0], stride = [1, 1]} : (tensor<5x32x32x5xf32>) -> tensor<5x30x30x5xf32>
-// CHECK-DAG:   "tosa.const"() {value = dense<[0, 3, 1, 2]> : tensor<4xi32>} : () -> tensor<4xi32>
-// CHECK-DAG:   %[[TRANS_MPOOL_RES:.*]] = "tosa.transpose"(%[[MPOOL_RES]], %3) : (tensor<5x30x30x5xf32>, tensor<4xi32>) -> tensor<5x5x30x30xf32>
+// CHECK-DAG:   "tosa.const"() {value = dense<[0, 3, 1, 2]> : tensor<4xi64>} : () -> tensor<4xi64>
+// CHECK-DAG:   %[[TRANS_MPOOL_RES:.*]] = "tosa.transpose"(%[[MPOOL_RES]], %3) : (tensor<5x30x30x5xf32>, tensor<4xi64>) -> tensor<5x5x30x30xf32>
```

Comment:
In TOSA they must be int32_t. Is tosa-mlir more relaxed?

Reply:
Yes, it is. I can also change this to `int32_t ` to match the standard